main.py

```diff
@@ -38,14 +38,19 @@ def main(input_code, input_lang, output_lang):
         # TODO: send 400 client error
         return "Error must specify output language. For example, js for javascript and py for python"
    
-    # don't return error without sending to database
-    output_code = ""
+
     if (type(gast) == str) :
-        # return error if gast not built
-        output_code = "Error: did not compile"
-    else:
-        output_code = gtc.gast_to_code(gast, output_lang)
+        # return error if gast not built - dont store in database
+        return "Error: did not compile"
```

Comment:
I'd prefer if there was a stronger link between this error response and the condition being checked in the data_collection code. Maybe a constant string or error code?